src/data/github-projects.json

```diff
@@ -193,6 +193,25 @@
             "repoURL": "https://github.com/badges/shields",
             "logo": "sheilds-io.svg"
         },
+        {
+            "title": "A-to-Z-Resources-for-Students",
+            "description": "Curated list of resources for college students.",
+            "tags": [
+                {
+                    "href": "https://twitter.com/diipakkr",
+                    "src": "https://img.shields.io/twitter/follow/diipakkr.svg?style=social",
+                    "alt": "Twitter Follow"
+                },
+                {
+                    "href": "https://github.com/dipakkr/A-to-Z-Resources-for-Students",
+                    "src": "https://img.shields.io/github/stars/dipakkr/A-to-Z-Resources-for-Students.svg?style=social&label=Star",
+                    "alt": "GitHub stars"
```

Comment:
Adding a `star` widget here doesn't make sense, because there is already a star widget by default (see your screenshot).